media/js/mozorg/manifesto.js

```diff
@@ -57,16 +57,30 @@ $(function() {
                         .text(window.trans('principle-nav-prev')).appendTo($nav);
 
                     $nav.on('click', 'button', function () {
-                        nav_modal($(this).hasClass('prev') ? -1 : 1);
-                    });
-
-                    window.dataLayer.push({
-                        event: 'manifesto-interaction',
-                        browserAction: 'modal open',
-                        section: section_id.match(/\d+/)[0]
+                        var $this = $(this);
+                        var quoteText = $('.principle.overlay-contents h3').text();
+                        // Track arrow clicks on Manifesto quote popup
+
+                        window.dataLayer.push({
+                            'event': 'manifesto-quote-click-arrow',
+                            'direction': ($this.attr('class') === 'prev') ? 'previous' : 'next',
+                            'quote': quoteText.slice(3),
+                            'quoteNumber': Number(quoteText.slice(0, 2))
+                        });
+
+                        nav_modal($this.hasClass('prev') ? -1 : 1);
                     });
                 }
             });
+
+            // Track Manifesto quote clicks
+
+            window.dataLayer.push({
+                'event': 'manifesto-quote-click',
+                'quote': $(this).find('h3').text().slice(3),
```

Comment:
Could use `$this` here instead? Also, same warning about localized text - this quote will be in different languages.